mk/config.mk

```diff
@@ -1063,3 +1063,11 @@ CFG_HMAC_64_1024_RANGE ?= n
 # By default use standard pbkdf2 implementation
 CFG_CRYPTO_HW_PBKDF2 ?= n
 $(eval $(call cfg-depends-all,CFG_CRYPTO_HW_PBKDF2,CFG_CRYPTO_PBKDF2))
+
+# Make other cores halt on panic() in a multi core setup. An SGI will be
+# triggered to notify all other CPUs in the same cluster that a panic() is
+# occurring. The SGI used to notify other cores is specified in:
+# CFG_CORE_HALT_CORES_ON_PANIC_SGI
+# Note: Other clusters of CPU won't be notified
+CFG_CORE_HALT_CORES_ON_PANIC ?= n
+CFG_CORE_HALT_CORES_ON_PANIC_SGI ?= 9
```

Comment:
I suggest to rename to `CFG_HALT_CORES_ON_PANIC` and `CFG_HALT_CORES_ON_PANIC_SGI`.

Reply:
When should we use CFG_CORE_XXX instead of CFG_XXX?

Reply:
> When should we use CFG_CORE_XXX instead of CFG_XXX?

I would say: when it is needed to make it clear that the feature relates to the TEE core. For example `CFG_CORE_HEAP_SIZE` or `CFG_CORE_ASLR` (vs. `CFG_TA_ASLR`). Otherwise it's better to have a shorter name. By that criterion, we probably have a few `CFG_CORE_XXX` too many, but not a huge number.

Reply:
I fully agree with all points stated by Jérôme's above comment.
I remember those old days when I was in favor of using `CFG_TEE_CORE_` prefix for all config switches related to OP-TEE core, and `CFG_TEE_TA_` for OP-TEE TA related configs. And I remember you said that `CFG_` was far enough in most case. I think you were right.

Reply:
Makes sense, thanks.

Reply:
I will shorten the switch, thanks for clarifying